docs/automated-process.md

```diff
@@ -11,6 +11,34 @@ release process. The automated processes uses GitHub Actions to automate the
     `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
     specify.
 
+## Setup for the release
+
+### Credentials
+
+- Access to https://github.com/ansible-community/ansible-build-data repo
+- Access to https://github.com/ansible/ansible repo
+- Access to Ansible Release Management working group (in Github).
+- Join Release Management working group and Community Working Group.
+- Be a part of the maintainer group in PyPI.
```

Comment:
I don't think PyPI access is strictly required anymore, unless something goes wrong and the process needs to be completed manually.

Reply:
I think the idea is to explicitly **not** have access to PyPI for most people.